daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -191,9 +191,8 @@ func subvolCreate(path, name string) error {
 		args.name[i] = C.char(c)
 	}
 
-	_, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_SUBVOL_CREATE,
-		uintptr(unsafe.Pointer(&args)))
-	if errno != 0 {
+	if _, _, errno := syscall.Syscall(syscall.SYS_IOCTL, getDirFd(dir), C.BTRFS_IOC_SUBVOL_CREATE,
```

Comment:
That is great. Thanks @ripcurld0 .
Changes had done, is it OK?
PTAL, @ripcurld0 @duglin @AkihiroSuda .